test/ctr.bats

```diff
@@ -719,7 +719,7 @@ function teardown() {
 	echo "$output"
 	[ "$status" -eq 0 ]
 	# Wait for container to OOM
-	run sleep 10
+	run sleep 100
```

Comment:
Should we add an echo "sleeping for 100 seconds to let container OOM" in case someone running from the command line is wondering what's what?

Reply:
Not something to hold for, maybe a touch up later.